zanata-client-commands/src/main/java/org/zanata/client/commands/pull/PullOptionsImpl.java

```diff
@@ -166,6 +167,16 @@ public boolean isContinueAfterError() {
     }
 
     @Override
+    public int getMinDocPercent() {
+        return this.minDocPercent;
+    }
+
+    @Option(name = "--min-doc-percent", aliases = "-m", usage = "When specified, will only pull down translation files that have translated or better statistics above the specified percentage")
```

Comment:
I assume "translated or better" means "Translated or Approved/Reviewed"?  It won't be clear to everyone.  Especially if they just try to read it as a sentence, because "translated" is really the name of a state in the system, but not everyone will know that.  Perhaps recast the sentence entirely, so that we don't have to talk about Translated or Approved at all.